docarray/typing/url/image_url.py

```diff
@@ -33,23 +31,6 @@ def _to_node_protobuf(self) -> NodeProto:
         """
         return NodeProto(image_url=str(self))
 
-    @classmethod
-    def validate(
-        cls: Type[T],
-        value: Union[T, np.ndarray, Any],
-        field: 'ModelField',
-        config: 'BaseConfig',
-    ) -> T:
-
-        url = super().validate(value, field, config)  # basic url validation
-        has_image_extension = any(url.endswith(ext) for ext in IMAGE_FILE_FORMATS)
-        if not has_image_extension:
-            raise ValueError(
-                f'Image URL must have one of the following extensions:'
-                f'{IMAGE_FILE_FORMATS}'
-            )
-        return cls(str(url), scheme=None)
-
```

Comment:
@dongxiang123 we cannot just remove this code, it is functional and needed. It validates that the image url actually points to a valid image file

Reply:
probably a mixup with removing the validate from any_url :thinking: lets bring back this function